doc/source/developer_guide/dg_about.rst

```diff
@@ -73,5 +73,11 @@ Overall, CICE code should be implemented as follows,
        call icepack_warnings_flush(nu_diag)
        if (icepack_warnings_aborted()) call abort_ice(error_message=subname, file=__FILE__, line=__LINE__)
 
+  * Use ``ice_check_nc`` or ``ice_pio_check`` after netcdf or pio calls to check for return errors.
```

Comment:
Should we make a note about setting/unsetting PIO_RETURN_ERROR around ice_pio_check too ? ice_pio_check won't do anything otherwise.

Reply:
Personally, I think this is a little too much detail to be in the documentation.  It's a tough balance between making the documentation useful versus overwhelming.  From my perspective, I hope folks will read the coding suggestions in the documentation (still not clear how often that happens), but also mostly follow current coding approaches.  If we want to add more details in the documentation, I'm open to that.  But we also need to maintain it.